src/main/scala/cromwell/binding/values/WdlFloatValue.scala

```diff
@@ -0,0 +1,28 @@
+package cromwell.binding.values
+
+import cromwell.binding.types.WdlFloatType
+
+case class WdlFloatValue(value: Float) extends WdlPrimitiveValue {
+  val wdlType = WdlFloatType
+  def add(rhs: WdlValue): WdlValue = {
+    rhs match {
+      case r:WdlFloatValue => WdlFloatValue(value + r.value)
```

Comment:
Yeah, miguel pointed out to me that this is all still very much in progress so i'll pipe down a bit :)